### DISCUSSION TITLE
OVER TIME with BETWEEN operator issue

### DISCUSSION THREAD

**Discussion by ShivaniSK** [finos/TimeBase-CE#221] [Q&A]
Hello,
We are trying to get data between specific start and end timestamp which is grouped over 1 minute.
The start and end timestamp range is given as **26th Aug 00:01 to 26th Aug 23:59**
E.g below is the sample query and expected output

`select last{} (value) from test over TIME (1m) where timestamp BETWEEN '2022-08-26 00:01:00'd AND '2022-08-26 23:59:00'd AND SYMBOL=='ABC'
`
Expected Output:
`

> 0,2022-08-26 20:57:00,ABC,0.99592
> 1,2022-08-26 20:58:00,ABC,0.99595
> 2,2022-08-26 20:59:00,ABC,0.99495
> 3,2022-08-26 21:00:00,ABC,0.99635
> 4,2022-08-26 21:01:00,ABC,0.99635

`

However, I am getting 1 extra record in the output for **28th August**. The date falls outside of the timerange which I have given in the BETWEEN clause.

Actual Output:
`

> 0,2022-08-26 20:57:00,ABC,0.99592
> 1,2022-08-26 20:58:00,ABC,0.99595
> 2,2022-08-26 20:59:00,ABC,0.99495
> 3,2022-08-26 21:00:00,ABC,0.99635
> 4,2022-08-26 21:01:00,ABC,0.99635
> **5,2022-08-28 18:51:00,ABC,0.99**

`

Can you please help us out on the issue.

**Comment by ankitsoin** [comment]
Hello, we fixed the issue by storing and retrieving data from spaces.
